src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,61 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int TYPE_OF_FIGURES_INDEX = 0;
+    private static final int FIGURES_INDEX = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] data = readFromFile(fromFileName);
+        int totalSupply = 0;
+        int totalBuy = 0;
+        for (String line : data) {
+            String[] splittedLine = line.split(",");
+            if (splittedLine[TYPE_OF_FIGURES_INDEX].equals("supply")) {
+                totalSupply += Integer.parseInt(splittedLine[FIGURES_INDEX]);
+            } else {
+                totalBuy += Integer.parseInt(splittedLine[FIGURES_INDEX]);
+            }
+        }
+        int totalResult = totalSupply - totalBuy;
+        String reportStatistic = "supply" + "," + totalSupply + System.lineSeparator()
+                + "buy" + "," + totalBuy + System.lineSeparator()
+                + "result" + "," + totalResult;
```

Comment:
lets also move logic of calculating buy and supply numbers in separate method

Reply:
appr separate methods were created